tests/units/signertest.cpp

```diff
@@ -20,7 +20,142 @@ void SignerTest::test() {
     testSigner();
 }
 
-void SignerTest::testSigner() {
+CopyrighFixer::Signature SignerTest::genSign(const QString licen,
+                                             const QString nOwn,
+                                             int tP) const {
+
+    CopyrighFixer::Owner ownerObj;
+    ownerObj.setName(nOwn);
+    ownerObj.setTimePoint(tP);
+
+    QMap<int, CopyrighFixer::Owner> OwnerMap;
+    OwnerMap.insert(tP, ownerObj);
+
+    CopyrighFixer::Signature signTest;
+    signTest.setLicenseTitle(licen);
+    signTest.setMessage("Distributed under the lgplv3 software license, see the accompany.");
+    signTest.setMapOwners(OwnerMap);
+
+    return signTest;
+}
+
+CopyrighFixer::Signature SignerTest::genSign(QList<SignerTest::dataOwns> numOwns, const QString licen) const {
+
+    QMap<int, CopyrighFixer::Owner> OwnerMap;
+    for (auto itemOwn: numOwns) {
+        CopyrighFixer::Owner ownerObj;
+        ownerObj.setName(itemOwn.name);
+        ownerObj.setTimePoint(itemOwn.timePoint);
+
+        OwnerMap.insert(itemOwn.timePoint, ownerObj);
+    }
+
+    CopyrighFixer::Signature signTest;
+    signTest.setLicenseTitle(licen);
+    signTest.setMessage("Distributed under the lgplv3 software license, see the accompany.");
+    signTest.setMapOwners(OwnerMap);
+
+    return signTest;
+}
+
+
+QList<SignerTest::Signers> SignerTest::lstSing() const {
+
+    // 1
+    // The case when the signatures in the config and the file match.
+    SignerTest::Signers equalSign;
+    equalSign.signFormConf = genSign("Copyright (C) 2020-2021 QuasarApp.", "QuasarApp", 1622657022);
```

Comment:
hm, i think the license title should be contains the license name. for example lgplv3 or MIT.